app/src/main/java/com/ecosystem/kin/app/MainActivity.java

```diff
@@ -54,6 +80,59 @@ public void onClick(View v) {
                 openKinMarketplace();
             }
         });
+
+        addNativeSpendOffer(nativeOffer);
+    }
+
+    // Use this method to remove the nativeOffer you added
+    private void removeNativeOffer(@NonNull NativeSpendOffer nativeSpendOffer) {
```

Comment:
not used

Reply:
it's for example, we are not adding a UI for that.